### DISCUSSION TITLE
Welcome to ezdxf Discussions!

### DISCUSSION THREAD

**Discussion by mozman** [mozman/ezdxf#275] [General]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AndreyB67** [comment]
Hello, my name is Andrey, I started working with autocad using the python library, of course there were questions, I hope to find answers to these questions from you, for my part, of course I will talk about interesting things in this area

**Comment by chibai** [comment]
Hello, my name is Huang, Yibin. I'm working in a floorplan-design startup in Shanghai, China.  The part of my work is recogizing the AutoCad file to floorplan, since RealDWG is really expensive, I use ezdxf to process dxf file after transfering dwg file to dxf file.  Majorly, I asked a lot of questions here to fix the problems in my work, and sometimes answered easy questions.

**Reply by AndreyB67** [reply]
Hello Huang. My name is Andrey. I am faced with the problem of extracting information from a dxf file concerning inscriptions on drawings. I can only output text in tables

**Reply by chibai** [reply]
I'm not sure what's your problem. As far as I know, there're 3 type entities related to text, MTEXT, TEXT and ATTRIB.

**Comment by AndreyB67** [comment]
hello, maybe I formulated it incorrectly today and considered the drawing, I brought out the entire Text content, but how do I find out their location in the drawing

**Reply by chibai** [reply]
> hello, maybe I formulated it incorrectly today and considered the drawing, I brought out the entire Text content, but how do I find out their location in the drawing

It's very hard to explain, but you can reference these 2 websites.
https://ezdxf.readthedocs.io/en/stable/dxfentities/text.html
https://ezdxf.readthedocs.io/en/stable/tutorials/text.html
And make full use of documentation of EZDXF
